#160703275 get food menu

nwashangai opened the pull request:
#### What does this PR do?
it adds get menu functionality
#### Description of Task to be completed?
* create an endpoint to get food menu
* write tests for this endpoint
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
[#160703275](https://www.pivotaltracker.com/story/show/160703275)
#### Screenshots (if appropriate)
* None
#### Questions:
* None

nwashangai committed:
ft: get food menu
    - create endpoint to get food menu
    - write test for this endpoint
    [Finishes #160703275]

dogoyaro commented on src/controllers/FoodController.js:
```diff
@@ -2,7 +2,16 @@ import FoodModel from '../models//FoodModel';
+            if (result.length > 0) 
+                response.status(200).json({ status: 'success', data: result });
+            else
+                response.status(200).json({ status: 'success', data: 'No food menu available' });
```
If the result is an array, I think it might be more useful for the case of no menu yet to be an empty array and not a string. That way it might be easier to manage the data on the front-end. This is just a suggestion though.

nwashangai committed:
ft: update order status
    - create endpiont to update status
    - write test cases for this endpoint
    [Finishes #160703227]

nwashangai committed:
ft: admin gets specific order
    - create endpoint to get specific order
    - write test cases for this endpoint
    [Finishes #160703172]

nwashangai committed:
ft: admin gets all orders
    - create endpoint to get all orders
    - write test cases for this endpoint
    [Finishes #160703159]

nwashangai committed:
ft: adds update menu item functionalities
    - create endpoint to update menu item
    - write test cases for this endpoint
    [Finishes #160703481]

nwashangai committed:
Merge pull request #42 from nwashangai/ft-update-menu-item-2197024

#160703481 adds update menu item functionalities

nwashangai committed:
Merge pull request #41 from nwashangai/ft-get-all-orders-2197024

#160703159 admin gets all orders

nwashangai committed:
Merge pull request #40 from nwashangai/ft-get-specific-2197024

#160703172 admin gets specific order

nwashangai committed:
Merge pull request #39 from nwashangai/ft-update-status-2197024
